src/main/java/Duke/Command.java

```diff
@@ -0,0 +1,142 @@
+package Duke;
+
+import common.Messages;
+import data.*;
+
+import static common.Messages.*;
+
+public class Command {
+    private Messages messageContainer = new Messages();
+    private String keyword;
+    private String[] tokenizedInput;
+    private String[] taskDescriptionRemarksFieldsInput;
+    private String queryTaskNumberText;
+
+    public Command(String keyword) {
+        this.keyword = keyword;
+    }
+
+    public Command(String keyword, String queryTaskNumberTextInput) {
+        this.keyword = keyword;
+        this.queryTaskNumberText = queryTaskNumberTextInput;
+    }
+
+    public Command(String keyword, String[] tokenizedInput, String[] processedUserInput) {
+        this.keyword = keyword;
+        this.tokenizedInput = tokenizedInput;
+        this.taskDescriptionRemarksFieldsInput = processedUserInput;
+    }
+
+    public void runCommand(TaskList taskListInput, Ui uiInput) throws DukeException {
+        Task newTask;
+        switch (keyword.toLowerCase()) {
+        case (BYE_COMMAND):
+            break;
+        case (LIST_COMMAND):
+            printTaskList(taskListInput, uiInput);
+            break;
+        case (DONE_COMMAND):
+            taskIsDone(taskListInput, uiInput, queryTaskNumberText);
+            break;
+        case (DELETE_COMMAND):
+            deleteTask(taskListInput, uiInput, queryTaskNumberText);
+            break;
+        default:
+            insertNewTask(taskListInput, uiInput, tokenizedInput);
+            break;
+        }
+    }
+    private void insertNewTask(TaskList listInput, Ui uiInput, String[] tokenizedInput) throws
+            DukeException{
+        Task newTask = null;
+        switch (tokenizedInput[0]) {
+        case (TODO_COMMAND):
+            newTask = new ToDo(taskDescriptionRemarksFieldsInput[0], taskDescriptionRemarksFieldsInput[1]);
+            break;
+        case (DEADLINE_COMMAND):
+            newTask = new Deadline(taskDescriptionRemarksFieldsInput[0], taskDescriptionRemarksFieldsInput[1]);
+            break;
+        case (EVENT_COMMAND):
+            newTask = new Event(taskDescriptionRemarksFieldsInput[0], taskDescriptionRemarksFieldsInput[1]);
+            break;
+        default:
+            uiInput.displayErrorMessage(INVALID_COMMAND_MESSAGE);
+            //break;
```

Comment:
Please remove this commented out code portions and all other commented out code portions